python-avd/pyavd/_eos_designs/structured_config/network_services/static_routes.py

```diff
@@ -50,6 +50,21 @@ def static_routes(self: AvdStructuredConfigNetworkServicesProtocol) -> None:
                     self.structured_config.static_routes.append_unique(static_route_item)
 
                 for svi in vrf.svis:
+                    for static_route in svi.static_routes:
+                        static_route_item = EosCliConfigGen.StaticRoutesItem()
+                        static_route_item._update(
+                            prefix=static_route.prefix,
+                            next_hop=static_route.next_hop,
+                            vrf=vrf.name,
```

Comment:
if the VRF is default wonder what it will render

Reply:
also for VRF default we need to check what happens. You should update `def _vrf_default_ipv4_static_routes` in network_services/utils.py to look for static routes in the SVIs